HelpAssistHer/help-assist-her
# large button class added to medium button component

## Description
- This button is similar to the Out of Business button but large insize
- The button will save progress for form
## Test Plan
- yarn webpack
- visit localhost/4000
- verify 
      1. save progress button will appear below the form button
      2. Click the button to save the form data 
      3. Hover over the button, it will turn orange
